src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/BailEmailNotificationPersonalisation.java

```diff
@@ -1,7 +1,28 @@
 package uk.gov.hmcts.reform.iacasenotificationsapi.domain.personalisation;
 
+import static uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.BailCaseFieldDefinition.DECISION_GRANTED_OR_REFUSED;
+import static uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.BailCaseFieldDefinition.IS_LEGALLY_REPRESENTED_FOR_FLAG;
+import static uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.BailCaseFieldDefinition.RECORD_THE_DECISION_LIST;
+import static uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.BailCaseFieldDefinition.SS_CONSENT_DECISION;
+
 import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.BailCase;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.ccd.field.YesOrNo;
 
 public interface BailEmailNotificationPersonalisation extends BaseNotificationPersonalisation<BailCase> {
 
+    default boolean isBailGranted(BailCase bailCase) {
+        return bailCase.read(DECISION_GRANTED_OR_REFUSED, String.class).orElse("").equals("granted")
+               || (bailCase.read(RECORD_THE_DECISION_LIST, String.class).orElse("").equals("Minded to grant")
```

Comment:
Can you check the ccd defn for the "Minded to grant" please.

Reply:
And also, as part of RIA-5520 there was a field populated to record the decision type in bail-case-api. src/main/java/uk/gov/hmcts/reform/bailcaseapi/domain/handlers/presubmit/DecisionTypeAppender.java.
Can you check if this can be simplified, just by reading the new field.